.github/workflows/test_pack_doc.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           versionTag: ${{ steps.prep.outputs.versionTag }}
         with:
           arguments: testReport
-      - name: Persist **Successfull** Java tests merged report
+      - name: Persist **Successful** Java tests merged report
```

Comment:
Thks, will merge later this week!

Reply:
Actually will probably change it to Passing instead